packages/fcl-wc/package.json

```diff
@@ -12,16 +12,6 @@
   "bugs": {
     "url": "https://github.com/onflow/fcl-js/issues"
   },
-  "jest": {
-    "transform": {
-      ".js": "jest-esm-transformer"
-    },
-    "testEnvironment": "./jest-test-environment.js",
```

Comment:
Do we need to add `"testEnvironment": "jsdom"` do packages?

Reply:
The default jest environment is sufficient now with the new version.